compiler/testData/diagnostics/tests/delegation/Delegation_MultipleDelegates.txt

```diff
@@ -9,7 +9,7 @@ internal trait One {
 }
 
 internal trait OneImpl : One {
-    invisible_fake open override /*1*/ /*fake_override*/ fun boo(): kotlin.Int
+    invisible_fake final override /*1*/ /*fake_override*/ fun boo(): kotlin.Int
```

Comment:
No, looks like I was wrong: the fakes are not final themselves, but simply inherit modality from parent, it's ok